mobile/library/swift/EnvoyCxxSwiftInterop/cxx_swift_interop.cc

```diff
@@ -0,0 +1,19 @@
+#import "cxx_swift_interop.h"
+#import "source/server/options_impl.h"
+#import "library/common/engine.h"
+
+namespace Envoy {
+namespace CxxSwift {
+
+void run(BootstrapPtr bootstrap_ptr, std::string log_level, envoy_engine_t engine_handle) {
+  auto options = std::make_unique<Envoy::OptionsImpl>();
+  std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> bootstrap(
+      reinterpret_cast<envoy::config::bootstrap::v3::Bootstrap*>(bootstrap_ptr));
```

Comment:
If you wished, I think you could do:
```
auto bootstrap = absl::WrapUnique(reinterpret_cast<envoy::config::bootstrap::v3::Bootstrap*>(bootstrap_ptr));
```
Or even just inline it:
```
options->setConfigProto(absl::WrapUnique(reinterpret_cast<envoy::config::bootstrap::v3::Bootstrap*>(bootstrap_ptr)));
```
But either way is fine.